src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo as TodoTypes } from '../../types/Todo';
+
+type Props = {
+  todo: TodoTypes,
+  deleteTodo: (id: number) => void,
+  Loader: boolean;
```

Comment:
```suggestion
  isLoading: boolean;
```
Boolean variables and etc should have a prefix `is` or `has` and it shouldn't be capitalized